content-warehouse/src/main/java/contentwarehouse/v1/CreateRuleSet.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contentwarehouse.v1;
+
+import com.google.cloud.contentwarehouse.v1.Action;
+import com.google.cloud.contentwarehouse.v1.ActionOrBuilder;
+import com.google.cloud.contentwarehouse.v1.CreateRuleSetRequest;
+import com.google.cloud.contentwarehouse.v1.CreateRuleSetRequestOrBuilder;
+import com.google.cloud.contentwarehouse.v1.DeleteDocumentAction;
+import com.google.cloud.contentwarehouse.v1.DeleteDocumentActionOrBuilder;
+import com.google.cloud.contentwarehouse.v1.ListRuleSetsRequest;
+import com.google.cloud.contentwarehouse.v1.ListRuleSetsRequestOrBuilder;
+import com.google.cloud.contentwarehouse.v1.LocationName;
+import com.google.cloud.contentwarehouse.v1.Rule;
+import com.google.cloud.contentwarehouse.v1.Rule.TriggerType;
+import com.google.cloud.contentwarehouse.v1.RuleOrBuilder;
+import com.google.cloud.contentwarehouse.v1.RuleSet;
+import com.google.cloud.contentwarehouse.v1.RuleSetOrBuilder;
+import com.google.cloud.contentwarehouse.v1.RuleSetServiceClient;
+import com.google.cloud.contentwarehouse.v1.RuleSetServiceClient.ListRuleSetsPagedResponse;
+import com.google.cloud.contentwarehouse.v1.RuleSetServiceSettings;
+import com.google.cloud.resourcemanager.v3.Project;
+import com.google.cloud.resourcemanager.v3.ProjectName;
+import com.google.cloud.resourcemanager.v3.ProjectsClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+
+// [START contentwarehouse_createruleset]
```

Comment:
please update sample region tag to match internal sample region tag database. see b/285617061#comment2.